### ISSUE TITLE
[Exercise] Capture multiple photos

### ISSUE THREAD

**Issue by xiao8579** [xiao8579/QuickCapture-Testing#39]
## Description
In 1.10, you can configure a button to captured multiple photos for the QuickCapture projects. Captured photos will be added as attachments to the captured record. User can add up to 5 photos for each record.

## Test 
1. Sign in to QuickCapture designer QA site.
1. Open an existing project, or deploy a new **Photo inspection** project from Esri templates.
1. Click on one of the buttons that capture the point record.
2. On the **DATA** tab, turn on **Take photo**.
3. Configure the number of photos to be taken by this button.
    - Configure a button that captures a minimum of 0 photos and a maximum of 3 photos.
    - Configure a button that captures a minimum of 2 photos and a maximum of 5 photos.
    - Configure a button that captures only 1 photo.
    - Configure a button that captures 4 photos.
4. Save the project when you're done.
5. Download the QuickCapture project to your device, take photos and submit the records.

## Feedback
1. What are the pain points you face while using it?
2. How can we improve the current workflow?
3. Were there any sluggish issues or crash issues? Describe what you saw.

**Comment by menzelc** [comment]
Android
Using BioBlitz Template, I changed the Ladybug observation to take 2-4 photos and I changed "Other" to take 2-3 photos.  Saved the project and refreshed my mobile project.
I click Ladybug and am able to SKIP adding photo.
I click Other and have to take 2-3, the app requires me to take at least 2 photos.

**Comment by IsmaelInRedlands** [comment]
@menzelc  Good catch. The expected behavior when you set a minimum number of photos is that the photos become required. We will need to fix this.

**Comment by jerjer007** [comment]
Issue: iPhone (12 Pro Max) is trying to display both the map and buttons side by side even though, the "Display on tablet" box is unchecked. I shared the project, Beach Debris, to the Survey123 community and to QQCreator.

**Comment by xiao8579** [comment]
iPhone 12 Pro Max

**Comment by Leandermen** [comment]
The multiple photo feature works fine for me using an iPad Air 2

**Comment by OCrypter** [comment]
1- What are the pain points you face while using it?
Ohan: I am lefty - so the preview is on the left side - but I am also the exception. 
2- How can we improve the current workflow?
Ohan: Maybe move the loading to the image button location
3- Were there any sluggish issues or crash issues? Describe what you saw.
Ohan: 2 issues : 

- 1. - multiple pictures can be taken while the limit is not hit. If you click quickly on the image capturing button, it will keep on taking pictures. Those pictures are based on where the camera is oriented, so if you move your hand after you clicked multiple times, looks like the algorithm will take the next picture as soon as the saving of the previous picture is over - till the last picture is taken
- 2. - My Token expired when taking multiple pictures - I was connected to WiFi. I had to exit the application and reload it to send the records. Here is the info 

> User: OOumoudian
> Device Info
> - Device model: iPhone 12 Pro
> - Qt version:5.15.2
> - AppFramework version: 5.1.80
> - Platform: ios
> - Operating system version: 14.6
> 
> App Info
> - App name: ArcGIS QuickCapture
> - App version: 1.10.154
> 
> Project Info:- Name: Photo Inspection - Holistic 
> - Last updated: 7/8/21
> - Owner: OOumoudian
> 
> Error Info
> - {"error":{"code":498,"details":["Invalid token."],"message":"Invalid token."}}

[Photo Inspection - Holistic -OOumoudian-20210708142938.zip](https://github.com/xiao8579/QuickCapture-Testing/files/6786587/Photo.Inspection.-.Holistic.-OOumoudian-20210708142938.zip)

**Comment by AlanaB8** [comment]
Had no issues with the multiple photo feature.

**Comment by kkalanta** [comment]
I defined the above scenarios in ArcGIS QuickCapture design and tested them in the mobile app. Everything is working well on my end. 
I tested the app on iPhone XS, version 15.2.1.
